tools/report-converter/codechecker_report_converter/markdownlint/output_parser.py

```diff
@@ -0,0 +1,59 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+import logging
+import os
+import re
+
+from ..output_parser import Message, BaseParser
+
+LOG = logging.getLogger('ReportConverter')
+
+
+class MarkdownlintParser(BaseParser):
+    """ Parser for Golint output. """
+
+    def __init__(self, analyzer_result):
+        super(MarkdownlintParser, self).__init__()
+
+        self.analyzer_result = analyzer_result
+
+        # Regex for parsing a clang-tidy message.
+        self.message_line_re = re.compile(
+            # File path followed by a ':'.
+            r'^(?P<path>[\S ]+?):'
+            # Line number followed by a ':'.
+            r'(?P<line>\d+?): '
+            # Checker name.
+            r'(?P<checker>[\S ]+) '
```

Comment:
Does space character need to be included in checker name? Doesn't it capture all words of the checker message eagerly except for the last one?